src/index.ts

```diff
@@ -42,6 +44,17 @@ export {
   flush,
 }
 
+export namespace CliUx {
+  export const ux = cliUx.ux
+  export const Config = cliUx.Config
+  export const config = cliUx.config
```

Comment:
What's the difference between `Config` and `config`? Do we need both of them?

Reply:
Config is the class and config is an instance of the class. See [config.ts](https://github.com/oclif/core/blob/phale/remove-cycle/src/cli-ux/config.ts#L58)